### ISSUE TITLE
Instruction to run this repo & data splits

### ISSUE THREAD

**Issue by kerkathy** [sig4kg/RAGAE#1]
Hi

I'm also interested in RAG research, specifically the retriever part.
I read your paper and want to have a try on this repo. Just wanna ask if you would provide any instruction on how to use this repo, maybe the finetuning/evaluation scripts, where're the datasets, how to train, inference, etc

Also, could you also provide the data split you used as train/dev/test set? Did you use random sampling to obtain these?

tks!

**Comment by sanianayab** [comment]
Following...
